packages/components/src/mobile/link-settings/test/edit.native.js

```diff
@@ -428,5 +428,147 @@ describe.each( [
 				}
 			);
 		} );
+
+		describe( 'Clear Button Behavior - Clear Button Presence', () => {
+			/**
+			 * GIVEN a LINK PICKER SHEET is displayed;
+			 * GIVEN the LINK PICKER field is FOCUSED but has NO text value;
+			 */
+			// eslint-disable-next-line jest/no-done-callback
+			it( 'should not display a CLEAR BUTTON within the LINK SETTINGS field', async ( done ) => {
```

Comment:
**Not Crazy Important:** The `done` callback usages will likely need to be removed when you update your branch because of recent changes related to `jest-circus`.
https://github.com/WordPress/gutenberg/pull/37715/files#r780551266

Reply:
@ttahmouch Thanks for the heads up! 👍